udata_gouvfr/views.py

```diff
@@ -292,3 +293,8 @@ def has_apis(ctx):
 def dataset_apis(ctx):
     dataset = ctx['dataset']
     return theme.render('dataset-apis.html', apis=dataset.extras.get(APIGOUVFR_EXTRAS_KEY))
+
+# TODO : better this, redirect is not the best. How to serve it instead ?!
```

Comment:
This is where I need some Python-fu : I want to add a `/stylemark` route that will serve the `stylemark/` directory which is located inside the theme static folder.
According to The Internet™, we should be using the `send_from_directory` function from Flask, but this require the corresponding static directory path which I do not know how to obtain.

Reply:
You could try to use `udata.theme.get_current_theme().static_path` but if the redirect solution works it does not look so bad.

Reply:
I'm going to let it as is for now, it works-ish but it's not ideal : you have to go to `/_stylemark/index.html` and you get redirected to an ugly URL. We want pretty URLs !